Image/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM circleci/python:3.7.0
```

Comment:
I think we should rename the folder. Image is usually reserved for docker images nowadays